core/domain/activity_jobs_one_off_test.py

```diff
@@ -2009,3 +2009,129 @@ def test_add_missing_skill_parent_model(self):
 
         self.assertIsNone(commit_model)
         self.assertItemsEqual(expected_output, actual_output)
+
+
+class SnapshotMetadataCommitMsgAuditOneOffJob(
```

Comment:
Please add tests for running the job twice, so that we are sure it can be rerun.

Reply:
done